Samples/GettingStarted/BinaryClassification_SentimentAnalysis/SentimentData.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.ML.Runtime.Api;
+
+namespace BinaryClassification_SentimentAnalysis
+{
+    public class SentimentData
+    {
+        [Column("0")] public string SentimentText;
```

Comment:
I suggest putting the attributes on separate lines as that's more in line with default formatting.